src/utils/copy-to-clipboard.js

```diff
@@ -0,0 +1,19 @@
+const copyToClipboard = (text, callback) => {
+    if (typeof window !== 'undefined') {
+      const hiddenInput = document.createElement('input');
+  
```

Comment:
(nit) please avoid having any empty lines within a function. The reason being that in many cases (including this one) the empty lines appear to be an attempt to convey some sort of meaning or separation of code; in which case: if your intention is to separate a group of lines, it would likely be better to just refactor into a separate, well-named function, if not, then please remove the empty lines. Either way: extra empty lines detract from clarity and readability.

Example:

```js
const getHiddenInputElement = () => {
  const result = document.createElement('input');
  result.type = 'text';
  result.style.height = '0px';
  result.style.opacity = '0';
  result.value = text;
  return result; 
};

const copyToClipboard = (text, callback) => {
  if (typeof window === 'undefined') { return; }
  const hiddenInputElement = getHiddenInputElement();
  document.body.appendChild(hiddenInputElement);
  hiddenInputElement.select();
  document.execCommand('copy');
  document.body.removeChild(hiddenInputElement);
  callback();  
};
```